mautic/mautic
# [UX] Tooltip to explain what is “Attribution” when creating a contact

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [x]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



### Description:
Since the market began to merge marketing automation tools with customer relationship tools, information related to attribution has gained a different definition from what is commonly thought of when we talk about marketing. 

In some cases, the attribution field could represent a monetary value of the lead, calculated using business data against the average purchase value of conversions made from the total number of leads. 

To avoid the confusion of potential users migrating platforms or something along those lines, a tooltip can benefit not only those who are marketing professionals but also for any entrepreneur who is implementing automation in their business without having the specialized human resource.



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open Contacts page
3. Click on New
4. Look at the Attribution fields

![image](https://github.com/mautic/mautic/assets/116097999/4d7819e6-1f8d-4c87-8b59-88d57395d701)

![image](https://github.com/mautic/mautic/assets/116097999/805acb89-7064-4b83-a6b0-8f6a876c1558)
